packages/utilities/src/utilities/settings/settings.js

```diff
@@ -11,11 +11,11 @@
  * @exports ibmdotcom.settings
  * @type {object} Settings object
  * @property {string} [prefix=dds]
- * Carbon for IBM.com v1.40.0',
+ * Carbon for IBM.com v1.41.1',
  *
  */
 const settings = {
-  version: 'Carbon for IBM.com v1.41.0',
+  version: 'Carbon for IBM.com v1.41.1',
```

Comment:
hmmm this is coming from the `main` branch, jeff did a test branch release that pushed some changes to `main`. Can you double check to make sure there's no carried no other carried over changes from `main`?

Reply:
I just [reverted the publish commit](https://github.com/andy-blum/carbon-for-ibm-dotcom/commit/7f58ec079748ee5585d55255d2aaa4c21724a969) - this should take care of these changes.